sw/supervision/python/parser.py

```diff
@@ -809,17 +809,12 @@ def __init__(self, conf_path):
 # [Data methods] Init data methods
 
     def init_data(self):
-        init = time.clock()
-
         self.load_cache()
         self.load_conf_files()
         self.load_sets_and_configs()
         self.load_devices()
         self.load_sessions_and_programs()
 
-        LOGGER.info("All data loaded in : %s seconds.\n",
```

Comment:
keep the log info without the time ?

Reply:
And get rid of the import time at the beginning of the file ?

Reply:
Done, and done.